frontend/src/lib/utils/eventUsageLogic.ts

```diff
@@ -902,5 +951,40 @@ export const eventUsageLogic = kea<
         reportPrimaryDashboardChanged: () => {
             posthog.capture('primary dashboard changed')
         },
+        reportDataManagementDefinitionClickView: ({ type, source }) => {
+            posthog.capture('definition click view', { type, source })
+        },
+        reportDataManagementDefinitionClickEdit: ({ type, source }) => {
+            posthog.capture('definition click edit', { type, source })
+        },
+        reportDataManagementDefinitionSave: ({ type, source }) => {
+            posthog.capture('definition saved', { type, source })
+        },
+        reportDataManagementDefinitionCancel: ({ type, source }) => {
+            posthog.capture('definition cancelled', { type, source })
+        },
+        reportDataManagementEventDefinitionsPageViewed: ({ loadTime, resultsLength, error }) => {
+            posthog.capture('event definitions page viewed', { load_time: loadTime, num_results: resultsLength, error })
+        },
+        reportDataManagementEventDefinitionsPageNestedPropertiesViewed: ({ loadTime, error }) => {
```

Comment:
Why do we track `num_results` on the event definitions, but not on the nested properties?

Reply:
I omitted this here since the number of nested properties will always be `<= PROPERTY_DEFINITIONS_PER_EVENT = 5`.